code/modules/projectiles/ammunition/ballistic/pistol.dm

```diff
@@ -48,3 +48,10 @@
 	caliber = ".50"
 	projectile_type = /obj/item/projectile/bullet/a50AE
 
+/obj/item/ammo_casing/caseless/lathe
+	name = "glass metal alloy round"
+	desc = "A round made out of a magnetized Glass and Metal alloy"
```

Comment:
desc = "A round made out of a magnetized glass and metal alloy." (full stop and remove capitals)